schema/workflow.yaml

```
@@ -1292,6 +1292,9 @@ $defs:
           - title: ExpressionDataSchema
             $ref: '#/$defs/runtimeExpression'
             description: An expression based event data schema.
+      data:
+        type: Object
+        description: Allow filtering by data content 
```

I would be consistent and add a title:

```suggestion
      data:
        type: Object
        title: EventData
        description: Allow filtering by data content 
```